examples/grafana/README.md

```diff
@@ -0,0 +1,27 @@
+# grafana-ocp
+
+This template creates a custom Grafana instance preconfigured to gather Prometheus openshift metrics.
+It is uses OAuth token to login openshift Prometheus.
+
+
+## To deploy grafana
+Note: make sure to have openshift prometheus deployed.
+(https://github.com/openshift/origin/tree/master/examples/prometheus)
+
+1. ```oc create namespace grafana```
+2. ```oc new-app -f grafana-ocp.yaml```
+3. grab the grafana url ``` oc get route |awk 'NR==2 {print $2}' ```
+4. grab the ocp token, from openshift master run: ```oc sa get-token prometheus -n kube-system```
+5. browse to grafana datasource's and add new prometheus datasource. 
+6. grab the prometheus url via ```oc get route -n kube-system prometheus |awk 'NR==2 {print $2}'``` and paste the prometheus url e.g https://prometheus-kube-system.apps.example.com
+6. paste the token string at the token field.
```

Comment:
There're two items "6"

Reply:
By the way, maybe the prometheus config in Grafana could be automated with a side-car container? Grafana has a REST API that we can use to add datasource. But this is only possible is prom URL and token can be found programmatically.

Just for information I did some POC for automatic grafana setup with prometheus in openshift, I don't know if it does have some interest for you: https://github.com/jotak/discomon
(The OS template: https://github.com/jotak/discomon/blob/master/prometheus-grafana-discovery.yml ). For POC I didn't care about securing access)

Reply:
@jotak yes we will drive grafana via api.  https://github.com/openshift/origin/pull/17114 is the start.

Reply:
done